locales/en/app.ftl

```diff
@@ -225,8 +228,8 @@ how-fxm-3-blurb = If you use {-brand-name}, you’ll receive a notification if y
   site that’s been breached. Find out right away if you were part of that breach 
   and what you can do about it.
 
-wtd-after-website = What to do after a website breach
-wtd-after-data-agg = What to do after a data aggregator breach
+wtd-after-website = What to do after a website breach:
```

Comment:
Do we care about other localizations picking up this change?

Reply:
What are your thoughts on this? If it doesn't change much in other locales, then I suppose not?

Reply:
Let's change it without new IDs. After all, they are supposed to look at the website and decide if it works or not for their language.